### PR TITLE
Feature/xs9 merge to master

### PR DESCRIPTION
**Pull request by liulinC**
Tested:
- Ring3: BVT+BST 200899 (Dev Run) Pass
- Repository Update Testing: 200902 (Dev Run) Pass (one unrelated failure)
- dummyrepoupdate.seq 4048674 Pass

### CODE REVIEW

File: python3/dnf_plugins/accesstoken.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+"""dnf plugin to set accesstoken http header for enabled repos"""
+import json
+import logging
+# Disable the error, it can be import in production env
+# and mocked out in unitttest
+# pylint: disable=import-
```
[the remark this answers is not in the document]
Comment (psafont) [reply python3/dnf_plugins/accesstoken.py:6]: This should be fixed

[the remark this answers is not in the document]
Comment (liulinC) [reply python3/dnf_plugins/accesstoken.py:6]: See https://github.com/xapi-project/xen-api/pull/5831

**Comment by liulinC** [comment]
Note: This PR should be merged together with xapi.spec repo

**Review by psafont — APPROVED** [review]

**Review by lindig — APPROVED** [review]
I'm not reviewing this in detail because it was reviewed previously.

**Pull request merged**